docs/en/02_Developer_Guides/01_Templates/02_Common_Variables.md

```diff
@@ -184,6 +184,14 @@ For instance, to only show the menu item linked if it's the current one:
 		<p>You are viewing the about us section</p>
 	<% end_if %>
 
+If you need alternative values for for "current" or "section" it's suggested you check against the `is` methods (`$isSection` and `$isCurrent`) and define you'r own value in the template:
+
+    :::ss
+    <a class="
+        <% if $isSection %>my-section-class<% end_if %>
+        <% if $isCurrent %>my-current-class<% end_if %>
+    " href="foo">Foo</a>
```

Comment:
Can you make this `" href="$Link">$MenuTitle</a>`

(sorry I should have included this in the original example)